src/ert/_c_wrappers/enkf/res_config.py

```diff
@@ -323,6 +326,118 @@ def _alloc_from_dict(self, config_dict):
                 config_file=config_file,
             )
 
+    def forward_model_job_name_list(self) -> List[str]:
+        return [j.name for j in self.forward_model_list]
+
+    @staticmethod
+    def write_jobs_json(path, forward_model_json):
+        with open(Path(path) / "jobs.json", mode="w", encoding="utf-8") as fptr:
+            json.dump(
+                forward_model_json,
+                fptr,
+            )
+
+    @staticmethod
+    def forward_model_data_to_json(
+        forward_model_list: List[ExtJob],
+        run_id: str,
+        data_root: str = "data_root",
+        iens: int = 0,
+        itr: int = 0,
+        context: "SubstitutionList" = SubstitutionList(),
+        env_varlist: Mapping[str, str] = None,
```

Comment:
The use of default arguments for dicts is a known cause for problems.
This is not readily applicable here, since we do not alter (at the time of writing) the inputs. Is there a better workaroud for this? None type is swapped later, on line 396:

```
    if env_varlist is None:
            env_varlist = {}
```